client-service-worker/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "dumber-gist",
+  "description": "Dumber gist client service worker",
+  "keywords": [],
+  "private": true,
+  "version": "0.0.1",
+  "license": "MIT",
+  "devDependencies": {
+    "cross-env": "^7.0.2",
+    "del": "^5.1.0",
+    "gulp": "^4.0.2",
+    "gulp-append-prepend": "^1.0.8",
+    "gulp-dumber": "^1.0.3",
+    "merge2": "^1.3.0"
+  },
+  "scripts": {
+    "start": "cross-env NODE_OPTIONS=--max_old_space_size=3584 gulp",
+    "build": "cross-env NODE_OPTIONS=--max_old_space_size=3584 gulp build",
+    "build:prod": "cross-env NODE_ENV=production NODE_OPTIONS=--max_old_space_size=3584 gulp build"
```

Comment:
max_old_space_size is not needed here. It was for bundling npm package `sass.js` in the bundler worker, the ecmascripted c-to-js code is too much for normal nodejs heap to handle.